samples/DataGridListSample.js

```diff
@@ -90,9 +91,11 @@ enyo.kind({
 	mixins: ["moon.SelectionOverlaySupport"],
 	selectionOverlayVerticalOffset: 35,
 	subCaption: "Sub Caption",
+	imageSizing: "cover",
```

Comment:
This new default is unnecessary. It changes the way that the sample is rendered, switching from `img` tag to `div` tag. It can stay only if we have a reason for the change.